src/modrover/learnerid.py

```diff
@@ -87,4 +87,7 @@ def __hash__(self) -> int:
         return hash(self.cov_ids)
 
     def __eq__(self, other) -> bool:
-        return hash(self) == hash(other)
\ No newline at end of file
+        return hash(self) == hash(other)
+
+    def __len__(self) -> int:
+        return len(self.cov_ids)
```

Comment:
Hahaha, in #8, I also want it to have `cov_ids` iterator, I think at this point we should revisit the directly inherit from `tuple` class idea :)

Reply:
Can make this a new ticket.

Reply:
Added to https://jira.ihme.washington.edu/browse/MSCA-222